requirements.txt

```diff
@@ -36,17 +29,12 @@ ipython-genutils==0.2.0
 jedi==0.16.0
 Jinja2==3.1.2
 jmespath==0.9.4
-jsondiff==1.1.2
-jsonpatch==1.24
-jsonpickle==1.2
-jsonpointer==2.0
-jsonschema==3.2.0
 kubernetes==26.1.0
 lockfile==0.12.2
 MarkupSafe==2.1.1
 matplotlib-inline==0.1.3
-mock==3.0.5
-moto==1.3.13
+more-itertools==10.5.0
+moto==2.0.8
```

Comment:
This gets us transact_write_items. There is probably value to going a bit further since this is still quite old and some of these older versions have older limits (e.g. a newer version has support for 100 items in a transaction, which we recently removed from Tron, though I'm sure there are other things :shrug:).